blscurve.nimble

```diff
@@ -25,7 +25,7 @@ proc test(env, path: string) =
   if not dirExists "build":
     mkDir "build"
   exec "nim " & lang & " " & env &
-    " --outdir:build -r --hints:off --warnings:off --skipParentCfg " & path
+    " --outdir:build -r -f --hints:off --warnings:off --skipParentCfg " & path
```

Comment:
why the `-f`?

Reply:
To force rebuilding while I tinker with C headers or Assembly code. It's good for you, even if you don't do that.

Reply:
I'm generally wary of running tests with different options than the "normal" build - ie if there's an issue with unforced builds, it would be good to bring it to the surface.

Reply:
This issue is old and known: the Nim compiler can't detect changes in all the source files that influence the resulting binary.

We can't enable "-f" by default everywhere, though, because it slows down recompilation significantly for huge projects like nimbus-eth2, so people got used to deleting "nimcache" when something's fishy.